pkg/cmd/adm/install_operator.go

```diff
@@ -0,0 +1,200 @@
+package adm
+
+import (
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/kubesaw/ksctl/pkg/client"
+	"github.com/kubesaw/ksctl/pkg/cmd/flags"
+	"github.com/kubesaw/ksctl/pkg/configuration"
+	clicontext "github.com/kubesaw/ksctl/pkg/context"
+	"github.com/kubesaw/ksctl/pkg/ioutils"
+	olmv1 "github.com/operator-framework/api/pkg/operators/v1"
+	olmv1alpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+	"k8s.io/client-go/util/homedir"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/spf13/cobra"
+)
+
+type installArgs struct {
+	kubeConfig string
+	namespace  string
+}
+
+func NewInstallOperatorCmd() *cobra.Command {
+	commandArgs := installArgs{}
+	cmd := &cobra.Command{
+		Use:   "install-operator <host|member> --kubeconfig <path/to/kubeconfig> --namespace <namespace>",
+		Short: "install kubesaw operator (host|member)",
+		Long:  `This command installs the latest stable versions of the kubesaw operator using OLM`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			term := ioutils.NewTerminal(cmd.InOrStdin, cmd.OutOrStdout)
+			ctx := newExtendedCommandContext(term, client.DefaultNewClientFromRestConfig)
+			return installOperator(ctx, commandArgs, args[0], time.Second*60)
+		},
+	}
+
+	defaultKubeConfigPath := ""
```

Comment:
`register-member` contains the same flag and handling code and I assume we're going to be introducing a lot more of it when we drop support for initializing the connection using just a token. Maybe it's a good time to introduce some utility function for this now?

Reply:
>  I assume we're going to be introducing a lot more of it when we drop support for initializing the connection using just a token

providing a kubeconfig instead of ksctl.yaml file would be just an alternative (optional flag), while in these cases it's a required. 
As mentioned in the other PR, I think that we should mark the `--kubeconfig` flag as required in those cases when we rely on that, so it's consistent and user always define the kubeconfig explicitly.
I made another PR to address that in the already existing commands https://github.com/kubesaw/ksctl/pull/57

Reply:
Done it is. I've made the kubeconfig required with no defaults in https://github.com/kubesaw/ksctl/pull/48/commits/d79bd2207532369d8548521e8b342cef94e38153

Reply:
I agree with @metlos we might want to handle this flags in a centralized way ( having some reusable function ) so that it's consistent across all the commands. We can introduce this later .